server/game/gamesteps/challenge/chooseAssaultTargets.js

```diff
@@ -0,0 +1,62 @@
+const BaseStep = require('../basestep.js');
+const AssaultKeywordAction = require('../../GameActions/AssaultkeywordAction.js');
+
+class ChooseAssaultTargets extends BaseStep {
```

Comment:
Really small thing, but the file name should be capitalized as well (ChooseAssaultTargets instead of chooseAssaultTargets)

Reply:
fixed